test/server/cards/02_SHD/units/TheMandalorian.spec.ts

```diff
@@ -0,0 +1,42 @@
+describe('The Mandalorian, Wherever I Go He Goes', function () {
+    integration(function (contextRef) {
+        describe('The Mandalorian\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
```

Comment:
Can you add some player2 units just to confirm that the filter is set up correctly so they are targetable (as it doesn't say "friendly unit" specifically)

Reply:
added